### DISCUSSION TITLE
[Script request] Docker Steam Headless LXC

### DISCUSSION THREAD

**Discussion by ptichalouf** [community-scripts/ProxmoxVE#170] [Request script]
### Application Name

Steam Headless

### Website

https://github.com/Steam-Headless/docker-steam-headless

### Description

Steam Headless is very important for me just because he work directly in docker so with lxc and we still can share the GPU to any other LXC and we can play steam big picture with sunshine or steam remote play. This will be a really good addition to all the other scripts.

Remote Game Streaming Server.

Play your games either in the browser with audio or via Steam Link or Moonlight. Play from another Steam Client with Steam Remote Play.

Easily deploy a Steam Docker instance in seconds.
Features:

    Steam Client configured for running on Linux with Proton
    Moonlight compatible server for easy remote desktop streaming
    One click installation of EmuDeck, Heroic and Lutris
    Full video/audio noVNC web access to a Xfce4 Desktop
    NVIDIA, AMD and Intel GPU support
    Full controller support
    Support for Flatpak and Appimage installation
    Root access
    Based on Debian Bookworm

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
Hello,

at the current state of the project we do not support game scripts or anything to do with games. There is no one who will maintain them and the problems will increase.

**Comment by kbftech** [comment]
Is steam considered as a game? I would argue that it's an app!

@ptichalouf There is a project doing it in docker, if that helps:
https://github.com/Steam-Headless/docker-steam-headless
